.github/workflows/test.yml

```diff
@@ -43,7 +43,7 @@ jobs:
         run: ct lint --all
 
       - name: Create kind cluster
-        uses: helm/kind-action@v1.0.0
+        uses: helm/kind-action@v1.2.0
```

Comment:
Not exactly sure what happened, but the test job was failing because the node in the kind cluster was never ready. Seems like it may have been a recent change to the GitHub runner environment

```
https://github.com/helm/kind-action/issues/42
```

```
https://github.com/actions/virtual-environments/issues/3673
```